src/clj/com/climate/claypoole.clj

```diff
@@ -68,7 +61,7 @@
 
   This is exposed as a public function because it's handy if you're
   instantiating your own ExecutorServices."
-  [& {:keys [daemon thread-priority] pool-name :name
+  [& {:keys [_daemon _thread-priority] _pool-name :name
```

Comment:
I don't think this does the same thing anymore. I think I had these keys present so that they would let users of this function see what arguments they needed to pass, without having to look at `impl/thread-factory`. Renaming the keys changes the API. I think, if you're doing this renaming, that you need to explicitly do the name mapping:

```clj
[& {_daemon :daemon _thread-priority :thread-priority _pool-name :name} :as args]
```

Reply:
You can disable lint warnings for unused destructured fn args in clj-kondo using:

``` clojure
{:linters {:unused-binding {:exclude-destructured-keys-in-fn-args true}}}
```

if you like.

Reply:
I will revert that and add linter ignore.
Thanks.

Reply:
Added:
```
#_{:clj-kondo/ignore [:unused-binding]}
(defn thread-factory
```